include/pisa/util/string_util.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
```

Comment:
Update `clang-format`

Reply:
This file is already in `util` dir so you may as well remove that `_util` from the file name.

I would prefer rather `algorithm/string.hpp` or something like that.